### PR TITLE
docs: add docstring for vse image encoder

### PR DESCRIPTION
**Pull request by bwanglzu**
Add docstring for VSE++ image encoder.

### CODE REVIEW

File: encoders/image/VSEImageEncoder/README.md [hunk 1]
```diff
@@ -1,14 +1,31 @@
...
+
+## Encode image with the encoder:
+
+Download model:
+
+```shell
+wget http://www.cs.toronto.edu/~faghri/vsepp/runs.tar && tar -xvf runs.tar
+```
+
+Encode an images:
```
Comment (JoanFM) [inline encoders/image/VSEImageEncoder/README.md:18]: plural or singular?

**Review by JoanFM — COMMENTED** [review]
We need an example on how to use it with the `docker` image as the rest of READMEs are doing

File: encoders/image/VSEImageEncoder/README.md (hunk 1 above)
Comment (davidbp) [reply encoders/image/VSEImageEncoder/README.md:18]: I would say... `Encode a image (or a batch of imgages)`

File: encoders/image/VSEImageEncoder/README.md [hunk 2]
```diff
@@ -1,14 +1,31 @@
...
+```python
+import numpy as np
+from PIL import Image
+
+from encoders.image.VSEImageEncoder import VSEImageEncoder
+
+img_path = '1.jpg' # load images
+img = Image.open(img_path)
+encoder = VSEImageEncoder(path='runs/f30k_vse++_vggfull/model_best.pth.tar')
+embedding = encoder.encode(image.unsqueeze(0).numpy())
```
Comment (davidbp) [inline encoders/image/VSEImageEncoder/README.md:29]: The conversion could be avoided with `im = imageio.imread(image_path)` which would load directly as np.ndarray. No worries now, we might take are in future updates.

File: encoders/image/VSEImageEncoder/manifest.yml [hunk 3]
```diff
@@ -2,13 +2,13 @@ manifest_version: 1
...
+    `VSEImageEncoder` encodes data from a ndarray, potentially BatchSize x (Channel x Height x Width) into a
+        ndarray of `BatchSize x D`. Taken from the paper implementation of (VSE++: Improving Visual-Semantic Embeddings with Hard Negatives)
     at https://github.com/fartashf/vseppauthor: Jina AI Dev-Team (dev-team@jina.ai)
 url: https://jina.ai
 vendor: Jina AI Limited
 documentation: https://github.com/jina-ai/jina-hub
-version: 0.0.4
+version: 0.0.5
 license: apache-2.0
 keywords: [Cross-Modal, Computer Vision, VSE, Hard Negative Mining, vsepp]
```
Comment (theUnkownName) [inline encoders/image/VSEImageEncoder/manifest.yml:13]: I think we need to include one of the types: audio, image, nlp, video.

Comment (bwanglzu) [reply encoders/image/VSEImageEncoder/manifest.yml:13]: ah good to know, thanks @theUnkownName !

File: encoders/image/VSEImageEncoder/README.md [hunk 4]
```diff
@@ -1,14 +1,68 @@
 # VSEImageEncoder
 
-`VSEImageEncoder` is the image Encoder used to extract Visual Semantic Embeddings. Taken from the results of
-VSE++: Improving Visual-Semantic Embeddings with Hard Negatives (https://arxiv.org/abs/1707.05612). This model
-extracts image feature embeddings that can be used in combination with a VSETextEncoder which in combination will
+`VSEImageEncoder` is the image Encoder used to extract Visual Semantic Embeddings.
```
Comment (JoanFM) [inline encoders/image/VSEImageEncoder/README.md:3]: is `an`

**Review by JoanFM — DISMISSED** [review]
Just this minor thing, is an encoder not THE encoder

**Review by JoanFM — APPROVED** [review]

**Pull request merged**